be/src/olap/rowset_builder.cpp

```diff
@@ -251,7 +251,9 @@ Status RowsetBuilder::wait_calc_delete_bitmap() {
 
 Status RowsetBuilder::commit_txn() {
     if (_tablet->enable_unique_key_merge_on_write() &&
-        config::enable_merge_on_write_correctness_check && _rowset->num_rows() != 0) {
+        config::enable_merge_on_write_correctness_check && _rowset->num_rows() != 0 &&
+        !(_tablet->tablet_state() == TABLET_NOTREADY &&
```

Comment:
Publish use the condition `_tablet->tablet_state() != TABLET_RUNING`, we'd better keep consistent with that?

Reply:
`Tablet::update_delete_bitmap`(for publish) and `RowsetBuilder::submit_calc_delete_bitmap_task`(for commit) use `_tablet->tablet_state() == TABLET_NOTREADY `